x-pack/legacy/plugins/uptime/public/components/functional/empty_state/empty_state_error.tsx

```diff
@@ -13,25 +13,45 @@ interface EmptyStateErrorProps {
   errors: GraphQLError[];
 }
 
-export const EmptyStateError = ({ errors }: EmptyStateErrorProps) => (
-  <EuiPanel>
-    <EuiEmptyPrompt
-      title={
-        <EuiTitle size="l">
-          <h3>
-            {i18n.translate('xpack.uptime.emptyStateError.title', {
-              defaultMessage: 'Error',
-            })}
-          </h3>
-        </EuiTitle>
-      }
-      body={
-        <Fragment>
-          {errors.map((error: GraphQLError) => (
-            <p key={error.message}>{error.message}</p>
-          ))}
-        </Fragment>
-      }
-    />
-  </EuiPanel>
-);
+export const EmptyStateError = ({ errors }: EmptyStateErrorProps) => {
+  const unAuthorized = errors.find(
+    (error: GraphQLError) => error.message && error.message.includes('unauthorized')
+  );
+
+  return (
+    <EuiFlexGroup justifyContent="center">
+      <EuiFlexItem grow={false}>
+        <EuiPanel>
+          <EuiEmptyPrompt
+            iconType="securityApp"
+            iconColor="subdued"
+            title={
+              <EuiTitle size="m">
+                {unAuthorized ? (
+                  <h3>
+                    {i18n.translate('xpack.uptime.emptyStateError.noPermission', {
+                      defaultMessage:
+                        "You don't have permission, please contact system administraion.",
```

Comment:
Can we change this to `Sorry, you are not authorized to view Uptime data.` or something like that?

We might also add `To access Uptime, you need read access to the index that contains your Heartbeat data.`

I'm not great with copy recommendations though, @elastic/kibana-docs can someone please give us feedback here?

Reply:
Thanks @gchaps ❤️

Reply:
i have opted for "You are not authorized to view Uptime data, please contact your system administrator."

thanks for the suggestions.